tests/test_app.py

```diff
@@ -280,25 +285,84 @@ def test_version_p(self) -> None:
-            assert os.path.exists(os.path.join(tmp_dir, "unittest_output_added.json"))
-            assert os.path.exists(os.path.join(tmp_dir, "unittest_output_deleted.json"))
+            assert os.path.exists(os.path.join(tmp_dir, f"{__name__}_added.json"))
+            assert os.path.exists(os.path.join(tmp_dir, f"{__name__}_deleted.json"))
 
     # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
 
     def test_patch_save_json_n(self) -> None:
         target_path = str(SAMPLES_DIR / "password.patch")
         _stdout, _stderr = self._m_credsweeper(["--diff_path", target_path, "--log", "silence"])
-        assert not os.path.exists(os.path.join(PROJECT_DIR, "unittest_output_added.json"))
-        assert not os.path.exists(os.path.join(PROJECT_DIR, "unittest_output_deleted.json"))
+        for root, dirs, files in os.walk(PROJECT_DIR):
+            self.assertIn("credsweeper", dirs)
+            for file in files:
+                self.assertFalse(file.endswith(".json"))
+            dirs.clear()
+
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+    def test_export_config_p(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            json_filename = os.path.join(tmp_dir, f"{__name__}.json")
+            _stdout, _stderr = self._m_credsweeper(["--export_config", json_filename, "--log", "silence"])
+            assert os.path.exists(json_filename)
+
+    # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+
+    def test_import_config_p(self) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            custom_config = os.path.join(tmp_dir, f"{__name__}.json")
+            shutil.copyfile(CREDSWEEPER_DIR / "secret" / "config.json", custom_config)
+            args = ["--config", custom_config, "--path", str(CREDSWEEPER_DIR), "--find-by-ext", "--log", "CRITICAL"]
+            _stdout, _stderr = self._m_credsweeper(args)
+            self.assertEqual("", _stderr.decode())
+            output = _stdout.decode()
+            self.assertNotIn("CRITICAL", output)
+            self.assertIn("Time Elapsed:", output)
+            self.assertIn("Detected Credentials: 0", output)
+            self.assertEqual(2, len(output.splitlines()))
+            # add .py to find by extension
+            modified_config = Util.json_load(custom_config)
+            self.assertIn("find_by_ext_list", modified_config.keys())
+            self.assertIsInstance(modified_config["find_by_ext_list"], list)
+            modified_config["find_by_ext_list"].append(".py")
+            Util.json_dump(modified_config, custom_config)
+            _stdout, _stderr = self._m_credsweeper(args)
+            output = _stdout.decode()
+            self.assertEqual("", _stderr.decode())
+            self.assertNotIn("CRITICAL", output)
+            self.assertIn("Detected Credentials:", output)
```

Comment:
If then I think below code block should be changed like below.

```suggestion
            self.assertIn("Time Elapsed:", output)
```